app/views/products/_form.html.erb

```diff
@@ -24,6 +24,13 @@
           <%= f.text_field :price, placeholder: "Price", autocomplete:"off",class:"form-control" %>
         </div>
       </div>
+      <div class="form-group">
+        <%= f.label :category %><br>
+        <% @categories.each do | select_caregory | %>
+          <%= f.radio_button( :category_id, select_caregory.id, checked: true) %>
```

Comment:
Why are you forcing `checked: true` here?  I think that can cause problems when you go to the Edit Product page because browsers will always show the last option as checked.  I guess that marking all options as checked will confuse your browser and it will choose the last one as the checked one.

Reply:
Done. 
![captura de pantalla 2019-01-03 a la s 5 20 39 p m](https://user-images.githubusercontent.com/43416022/50664708-fd811500-0f7b-11e9-9750-c7ef0b23b493.png)